### ISSUE TITLE
[MNG-6138] Remove obsolete message_*.properties form maven-core

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#7304]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** opened **[MNG-6138](https://issues.apache.org/jira/browse/MNG-6138?redirect=false)** and commented

The folowing files are not used in the entire project tree:

```
$ find . -name '*.properties'
./src/main/resources/org/apache/maven/messages/messages.properties
./src/main/resources/org/apache/maven/messages/messages_de.properties
./src/main/resources/org/apache/maven/messages/messages_el.properties
./src/main/resources/org/apache/maven/messages/messages_en.properties
./src/main/resources/org/apache/maven/messages/messages_es.properties
./src/main/resources/org/apache/maven/messages/messages_fr.properties
./src/main/resources/org/apache/maven/messages/messages_ja.properties
./src/main/resources/org/apache/maven/messages/messages_ko.properties
./src/main/resources/org/apache/maven/messages/messages_nl.properties
./src/main/resources/org/apache/maven/messages/messages_no.properties
./src/main/resources/org/apache/maven/messages/messages_pl.properties
./src/main/resources/org/apache/maven/messages/messages_zh_CN.properties
```

The have been added years ago, but never used. Since we do not strive for any i18n, they are safe to be removed.

---

**Affects:** 3.3.9

**Remote Links:**
- [Page
](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=67636510)

**Backported to:** [3.5.0-alpha-1](https://github.com/apache/maven/milestone/92?closed=1)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [b5b82988a76d311e3ff94b29f2d05d2cb9062ffc](https://git-wip-us.apache.org/repos/asf?p=maven.git;a=commit;h=b5b82988a76d311e3ff94b29f2d05d2cb9062ffc).

**Comment by jira-importer** [comment]
**[Stephen Connolly](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=stephenc)** commented

Maven 3.4.0 has been dropped. See [this thread](https://mail-archives.apache.org/mod_mbox/maven-dev/201701.mbox/%3CCA%2BnPnMx-e7kGYy3Hp87v8hLGdhp1q%3DtKLx_6QuZ4kGUqHEBGcw%40mail.gmail.com%3E) for more details.

This issue will need to be re-scheduled for a Maven release in the (hopefully near) future.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [8373c79a47ce4f12e549ae817a9c5f9eae950fb8](https://git-wip-us.apache.org/repos/asf?p=maven.git;a=commit;h=8373c79a47ce4f12e549ae817a9c5f9eae950fb8).
